.travis.yml

```diff
@@ -317,6 +317,13 @@ jobs:
           - USE_MAVEN_REPO="true"
           - CMD="./.ci/travis/travis.sh no-error-pmd"
 
+    # No error testing - josm
```

Comment:
`No violation testing - josm (ant task)`

the same for DESC below.

because build is failed even if there is WARN violations.

Reply:
done

Reply:
DESC is not updated.

Reply:
done